app/code/Magento/ReviewGraphQl/Model/ResourceModel/GetProductRatingSummary.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ReviewGraphQl\Model\ResourceModel;
+
+use Magento\Framework\App\ResourceConnection;
+use Magento\Downloadable\Model\Link\Purchased\Item;
+
+/**
+ * The model returns the product rating summary
+ */
+class GetProductRatingSummary
```

Comment:
I would suggest moving this class to the `Review` module. According to the latest conversation with @mslabko, it's better to keep domain logic in domain modules.

So, we may move this model to the `Review` module (but do not put it into `resourceModel`, just to Model and maybe some subdir). We may also create an interface for this service but do not mark it as `@api`.

Reply:
Moved ✅